Талос. Опять

Chituka opened the pull request:
Чейнджлог дофикса фикса Талоса:
1. Расширен мостик. Теперь выглядит посвободнее
2. Переместили местами контейнер с кровью и контейнер с бадибагами, пофиксили прикол с 8 бесплатными аптечками Талосу из-за ошибки прошлого маппера
3. Пофиксили декали от прошлого реворка.
4. Убрали вендор с алкоголем у мастера эт эрмс. No more ИКОТА.
5. Пофиксили ошибки стилистические на кухне и сделали комнату мастера в более тёмных тонах
6. Переделали двигатели для того, чтобы их можно было починить при аварии
7. Переделали трубы вокруг тэга, чтобы можно было починить после аварии
8. Добавили вторую обычную аптечку и переместили их в ящики санитаров
9. Мелкие дофиксы
10. Добавили нормальный ящик в стенке с магбутами в бриге.

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.

Chituka committed:
Update inteq_talos.dmm

И снова мы обновляем Талос. Всё никак не остановимся.

Chituka committed:
Update inteq_talos.dmm

ДРАКОША ЗАБЫЛ

Erring commented:
Пока смотрел на локалке, понял ошибку одну, связанную с стационарным портом, **лучше не мержить до нового коммита**. Я немного с портом ошибся (высота слишком мелкой была, как и изменение расстояния от центра), но по идее с новым коммитом станет лучше, коммит будет завтра (уже сегодня) днем примерно, ибо у меня гит что-то не хочет делать хоть что-то...

Chituka committed:
Дофиксы... Again

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/b983bbaa-8930-4bc6-8265-0b5d6c886264)
Для чего эта труба?

molnizz commented:
@Chituka

Erring commented:
> ![image](https://private-user-images.githubusercontent.com/161384279/329778280-b983bbaa-8930-4bc6-8265-0b5d6c886264.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU0Mjk4MTQsIm5iZiI6MTcxNTQyOTUxNCwicGF0aCI6Ii8xNjEzODQyNzkvMzI5Nzc4MjgwLWI5ODNiYmFhLTg5MzAtNGJjNi04MjY1LTBiNWQ2Yzg4NjI2NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMVQxMjExNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZDk3MDRlYjQ2MTA2Y2Q5NjU1ZWJhZTAzZTAxYmJjNzI0ZWMwY2E4NTM2OGZhYjE5ZTI5NGI1NDIzNzI3ZDljJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CVgKB_uwWG1RGp507CmVR2T3E8H1M8xZdsOL3OAJ8DY) Для чего эта труба?

Оранжевая для закачки из канистр в камеры с газом. Вполне рабочей на локалке была.
Помпа же для закачки плазмы в сеть. Помпа вполне рабочая, просто трубу не показывает дальше, ибо та спрятана под полом.
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/44509388/1c141dae-f893-4b28-afb2-ded24fcf971d)

molnizz commented:
Я отсюда вижу, что до верхнего движка не достаёт провода

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/cb28c05b-e6a7-416b-983e-e5951a73ebf8)
Двойной терминал

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/0ae08f49-12ca-47d6-bad6-636c45177f44)

molnizz commented:
Хочу корнер здесь 
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/df2ac8a8-2502-462f-b4b6-6b6e37c0755d)

Erring committed:
Update inteq_talos.dmm

Добавил углы (на оригинальном талосе их нет, но пусть будут, ладно...)
Убрал летающий файр алярм и добавил кабели двигателю. It should work now.

Erring committed:
Update inteq_talos.dmm

Прошлый талос удалился, делал по памяти изменения, забыл терминал, починил стенку (опять), теперь всё готово.

Chituka commented:
@molnizz

Erring committed:
Update inteq_talos.dmm

Erring committed:
Вернул консоль, убранную самым первым реворком

Erring committed:
QoL труб и переставил контроллер турелей

molnizz commented:
У талоса в этой части есть шкафчик 
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/8a223901-e8eb-4ea9-868b-011b6fa36c3a)
Нужно выставить параметр icon_door = NULL

Erring commented:
> У талоса в этой части есть шкафчик без двери ![image](https://private-user-images.githubusercontent.com/161384279/329809038-8a223901-e8eb-4ea9-868b-011b6fa36c3a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MjE2MjcsIm5iZiI6MTcxNTUyMTMyNywicGF0aCI6Ii8xNjEzODQyNzkvMzI5ODA5MDM4LThhMjIzOTAxLWU4ZWItNGVhOS04NjhiLTAxMWI2ZmEzNmMzYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQxMzQyMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZTA5YjczYzdmMmFlMTk3YWI5Y2QwNDcyNmIwOGIyNjg3ODlmNWM3ZmI1ZDQ0NmM5MTFjM2VkM2MzYjhhODVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0Dw6k87cxKnKVNjR1MYslYKoOAjNVejDJVNHRJJ_A3s) Нужно выставить параметр icon_door = NULL

Это у всех кораблей с такими шкафчиками так по какой-то причине. Проблема не мапперов, а кодеров.

Erring commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/44509388/eb3b47e0-f6b2-4e2e-a40f-37dde082f7c6)
И да, это на старой версии до реворка. На новой версии этих ящиков просто нет

molnizz commented:
> > У талоса в этой части есть шкафчик без двери ![image](https://private-user-images.githubusercontent.com/161384279/329809038-8a223901-e8eb-4ea9-868b-011b6fa36c3a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MjE2MjcsIm5iZiI6MTcxNTUyMTMyNywicGF0aCI6Ii8xNjEzODQyNzkvMzI5ODA5MDM4LThhMjIzOTAxLWU4ZWItNGVhOS04NjhiLTAxMWI2ZmEzNmMzYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQxMzQyMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZTA5YjczYzdmMmFlMTk3YWI5Y2QwNDcyNmIwOGIyNjg3ODlmNWM3ZmI1ZDQ0NmM5MTFjM2VkM2MzYjhhODVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0Dw6k87cxKnKVNjR1MYslYKoOAjNVejDJVNHRJJ_A3s) Нужно выставить параметр icon_door = NULL
> 
> Это у всех кораблей с такими шкафчиками так по какой-то причине. Проблема не мапперов, а кодеров.

Это как раз таки проблема мапперов, потому что после залития фикса от офоф наши шкафы не починились, но если нет шкафчика то хорошо

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/a6baf274-6873-4ffe-aa03-6ea19a435646)
А как заряжать ???

molnizz commented:
Хочу чтобы было так:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/1e9e6f76-a2fa-47cf-b18e-b6f307df39a6)
Но оно вот так:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/627000b0-6154-4eae-8e35-131e115abcd5)

molnizz commented:
Меняй игральные карты на другие:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/7c954731-197d-46c4-ab60-5ba23ea1ec86)

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/d9a2b8e7-6205-42d8-a6ea-26034df02cde)
Убираешь Ammo Can и меняешь их на три коробки для дробовика и две коробки для пистолета

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/23bfc803-6757-4b0e-a6f5-f40f58151a82)
Убираешь их

molnizz commented:
Вырезаешь это:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/313c2112-e1c5-4b06-bfab-0afb7629dd8b)

molnizz commented:
Оставляешь 8 радио:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/bbbcea52-62f2-4ca3-8796-32ac38ca3688)

molnizz commented:
У инженеров убираешь двойные и меняешь на одинарные:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/39623f10-760b-47c6-b851-a5680c91d0e7)

molnizz commented:
Оставляешь три танка для кислорода:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/c7cc61d6-5336-4228-a2bd-3328706324d5)

molnizz commented:
Тут тоже оставляешь три танка:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/0b3cefb3-eaa1-4b5b-8a25-132112f234f9)

molnizz commented:
Слишком много /obj/item/tank/internals/oxygen в шкафчиках:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/daed4f8c-9da4-4e19-a5d4-4df2ae2c5a1f)
Урезай

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/cbb5eafb-5f6f-4c46-9de5-523967b30cf2)
Сделай его подпольным

molnizz commented:
Сделай всё сглаженным:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/50365a5e-f998-40d0-a4a1-31da1e475595)

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/8f3f29a7-3408-4f77-8fe4-872e53759cb2)
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/8091512a-9954-4688-b1c0-e6185b1524d8)
Убрать шторки

molnizz commented:
У тебя идут провода к этому окну:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/b8273b50-e304-4d55-bb44-c029986c9a56)
Но не идут к этому:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/2d15b669-fb66-4fd1-84bf-36105d65b414)

molnizz commented:
Раз нет гриля: 
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/611c0ba9-0770-48a6-b74a-54b818204e32)
Два нет гриля:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/c184beaa-508d-43cc-8418-fc0f444eebbf)
Три:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/04d3c251-faed-4d7d-9a3b-e3eb3a7fa93e)
4:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/59665bb7-2eb3-40c8-99d2-3dbf1474afca)
5:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/f2e1a670-399f-4187-8a1d-9f097e947e89)
Интересно, что он тут запитывает:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/51b72837-5fca-4585-bc55-38ad633cb6fb)
Дверь в окне:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/bd3c7220-e6f7-4f69-811d-34b42d7dccfc)
8:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/790595f2-628d-44f8-b560-64df207bfa0d)
Дальше сам

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/d1da6288-e261-43c6-85f9-7cceece197d6)

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/9897e247-ebe7-41e6-9fc2-a8fd0ed8582f)

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/23eb3c5f-5ead-4f58-81ec-21aa61608fba)

molnizz commented:
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/1e917e5a-fce7-4843-bb63-798f6dff0784)
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/161384279/c9ea12be-78f3-4e83-a083-9ca3969c2643)
Выставляй фракционные листы на турелях:

Erring commented:
> Меняй игральные карты на другие: ![image](https://private-user-images.githubusercontent.com/161384279/329878414-7c954731-197d-46c4-ab60-5ba23ea1ec86.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NTAzMDQsIm5iZiI6MTcxNTU1MDAwNCwicGF0aCI6Ii8xNjEzODQyNzkvMzI5ODc4NDE0LTdjOTU0NzMxLTE5N2QtNDZjNC1hYjYwLTViYTIzZWExZWM4Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQyMTQwMDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iODA1NzNkNTFmYjBjY2JiNDE2ZjJjMDg3NjgxMTYzMDZmMDVmODJlNzc5ZTM4YWJjZGU2Y2I2YjdmMDBiZTQyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._uaTmMhlOskzLKXIAJIr9ENRJIvtkxKkQsGshT5uizA)

Фича с оригинального талоса, зачем мне трогать?
Ты последнюю версию не смотришь, а потом смотришь... Оценивай именно последнюю версию. Настоятельно рекомендую посмотреть на оригинальную версию пред этим. 
Спавнера окон создают ещё решётки, а провода питают решётки под ним (защита базовая от людей без изолек). Дверь в окне пофиксил. Провода тоже

Турели не трогал, они так и были как на реворке первом (их поставил бурбончик), так и оставил. Фракций не было вообще в оригинальном талосе, так зачем их ставить? Убрал фракции вообще, ибо в hackmd.io от оффов про строительство шаттлов это не упоминается вообще.

> ![image](https://private-user-images.githubusercontent.com/161384279/329879807-cbb5eafb-5f6f-4c46-9de5-523967b30cf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NTAzMDQsIm5iZiI6MTcxNTU1MDAwNCwicGF0aCI6Ii8xNjEzODQyNzkvMzI5ODc5ODA3LWNiYjVlYWZiLTVmNmYtNGM0Ni05ZGU1LTUyMzk2N2IzMGNmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQyMTQwMDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MGRhMmRlN2U0YzA2Mzc3MTc5MTdkNWIzYjBjMzkwMGU0ZDg0NTU2Mjg1MDA4OWE4Y2MzNmNlOGI4ZWJiNTIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.y0yUQOg1IqVxHxGjhf7vaQAlukRWC9ITZKHC-puG1q8) Сделай его подпольным


Фича изначально исходит из первого реворка. Изначально я сделал так, как вы просили, но Бурбончик переделал ее, чтобы сделать более понятной, куда она ведет.



Сглаживать двойной нос не буду. Это фича с оригинального талоса.

Количество баллонов больших с кислородов осталось с талоса до реворка. Там 6 баллонов обычных кислородных и 2 плазменных в каждом ящике.

Я сейчас вообще уберу углы варнингов, они плохо смотрятся по итогу, ибо они меньше чем тёмно-желтая линия.
![image](https://github.com/MysticalFaceLesS/Shiptest/assets/44509388/ed57c727-03df-477b-834c-3025b457fcd3)

Зачем менять количество наушников, если это должно быть фичей? Распределять всё равно нет смысла и их всё равно столько (если считать наушники в других местах), сколько и экипажа. Это и удобно, и позволяет восстановить часть эквипа если кодеры напортачат опять со снаряжением.

Двойные пусть останутся фичей, ибо их никто не использует, они хорошие, да, но их можно спокойно сделать и в протолате сразу же почти. Хоть они и меньше, чем карманный плазма танк (тот имеет воздуха на буквально час дыхания, а плюоксиума на все 5-6).

Спасибо за проверку, @molnizz! Нашли сюрпризы, хоть они и особо не страшны.

Erring committed:
Update inteq_talos.dmm

molnizz commented:
> > Меняй игральные карты на другие: ![image](https://private-user-images.githubusercontent.com/161384279/329878414-7c954731-197d-46c4-ab60-5ba23ea1ec86.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NTAzMDQsIm5iZiI6MTcxNTU1MDAwNCwicGF0aCI6Ii8xNjEzODQyNzkvMzI5ODc4NDE0LTdjOTU0NzMxLTE5N2QtNDZjNC1hYjYwLTViYTIzZWExZWM4Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQyMTQwMDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iODA1NzNkNTFmYjBjY2JiNDE2ZjJjMDg3NjgxMTYzMDZmMDVmODJlNzc5ZTM4YWJjZGU2Y2I2YjdmMDBiZTQyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._uaTmMhlOskzLKXIAJIr9ENRJIvtkxKkQsGshT5uizA)
> 
> Фича с оригинального талоса, зачем мне трогать? Ты последнюю версию не смотришь, а потом смотришь... Оценивай именно последнюю версию. Настоятельно рекомендую посмотреть на оригинальную версию пред этим. Спавнера окон создают ещё решётки, а провода питают решётки под ним (защита базовая от людей без изолек). Дверь в окне пофиксил. Провода тоже
> 
> Турели не трогал, они так и были как на реворке первом (их поставил бурбончик), так и оставил. Фракций не было вообще в оригинальном талосе, так зачем их ставить? Убрал фракции вообще, ибо в hackmd.io от оффов про строительство шаттлов это не упоминается вообще.
> 
> > ![image](https://private-user-images.githubusercontent.com/161384279/329879807-cbb5eafb-5f6f-4c46-9de5-523967b30cf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NTAzMDQsIm5iZiI6MTcxNTU1MDAwNCwicGF0aCI6Ii8xNjEzODQyNzkvMzI5ODc5ODA3LWNiYjVlYWZiLTVmNmYtNGM0Ni05ZGU1LTUyMzk2N2IzMGNmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQyMTQwMDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00MGRhMmRlN2U0YzA2Mzc3MTc5MTdkNWIzYjBjMzkwMGU0ZDg0NTU2Mjg1MDA4OWE4Y2MzNmNlOGI4ZWJiNTIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.y0yUQOg1IqVxHxGjhf7vaQAlukRWC9ITZKHC-puG1q8) Сделай его подпольным
> 
> Фича изначально исходит из первого реворка. Изначально я сделал так, как вы просили, но Бурбончик переделал ее, чтобы сделать более понятной, куда она ведет.
> 
> Сглаживать двойной нос не буду. Это фича с оригинального талоса.
> 
> Количество баллонов больших с кислородов осталось с талоса до реворка. Там 6 баллонов обычных кислородных и 2 плазменных в каждом ящике.
> 
> Я сейчас вообще уберу углы варнингов, они плохо смотрятся по итогу, ибо они меньше чем тёмно-желтая линия. ![image](https://private-user-images.githubusercontent.com/44509388/329882809-ed57c727-03df-477b-834c-3025b457fcd3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NTIwODksIm5iZiI6MTcxNTU1MTc4OSwicGF0aCI6Ii80NDUwOTM4OC8zMjk4ODI4MDktZWQ1N2M3MjctMDNkZi00NzdiLTgzNGMtMzAyNWI0NTdmY2QzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEyVDIyMDk0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkMWExM2IwMDU3ZWU2ZWVlNjhlNGQ5Yjc3NjlmYTc5N2U0MjMxODc3MGVkMWE2YmI5NWQ1NTNhZWFhYmRkZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MzvMOp1XARhKNmUw6fpX-xiHqCPInapwa4CplJHxQVc)
> 
> Зачем менять количество наушников, если это должно быть фичей? Распределять всё равно нет смысла и их всё равно столько (если считать наушники в других местах), сколько и экипажа. Это и удобно, и позволяет восстановить часть эквипа если кодеры напортачат опять со снаряжением.
> 
> Двойные пусть останутся фичей, ибо их никто не использует, они хорошие, да, но их можно спокойно сделать и в протолате сразу же почти. Хоть они и меньше, чем карманный плазма танк (тот имеет воздуха на буквально час дыхания, а плюоксиума на все 5-6).
> 
> Спасибо за проверку, @molnizz! Нашли сюрпризы, хоть они и особо не страшны.

Очень тяжело просматривать последнюю версию, потому что, то что я смотрел, я скачивал с указанной ветки гита.

Игральные карты синдиката не очень вяжутся на корабле интека.

Если баллоны можно сделать на протолате, то пусть их и делают в протолате, а оставить только в шкафчике СЕ

Очень некрасиво выглядит выпирающая труба, поэтому и прошу убрать её под пол

Erring commented:
> Очень тяжело просматривать последнюю версию, потому что, то что я смотрел, я скачивал с указанной ветки гита.
Скачивай с последнего коммита
> Игральные карты синдиката не очень вяжутся на корабле интека.
Игральные карты это контрабанда, которая широко распространена, ибо его даже на обычной станции в 13 можно было найти.
> Если баллоны можно сделать на протолате, то пусть их и делают в протолате, а оставить только в шкафчике СЕ
То, что их можно сделать не означает то,
> Очень некрасиво выглядит выпирающая труба, поэтому и прошу убрать её под пол и я прошу убрать трубу не внутри, а снаружи
Мммм, ладно, под окном плохо видно, но всё же видно трубы. Поменял ещё название скрабберам, пусть будут жить.
> Наушники убрать, потому что они и так спавнятся с ними
Фича оригинального талоса + это добавляет возможность вербовать новых + защита от ошибок кодеров (again)

Erring committed:
Update inteq_talos.dmm

Erring committed:
Update inteq_talos.dmm

Erring committed:
Возможный фикс цикла внутри (всё ещё придется следовать инструкции, но развинчивать не придется.

Chituka commented:
@molnizz проверяй

molnizz commented:
У меня когда время появляется, тогда я и проверяю
